lib/usecases/get_survey_submission_use_case.dart

```diff
@@ -0,0 +1,29 @@
+import 'dart:convert';
+import 'package:injectable/injectable.dart';
+import 'package:survey_flutter_ic/api/storage/storage.dart';
+import 'package:survey_flutter_ic/model/survey_submission_model.dart';
+import 'package:survey_flutter_ic/usecases/base/base_use_case.dart';
+
+@Injectable()
+class GetSurveySubmissionUseCase
+    extends NoParamsUseCase<SurveySubmissionModel?> {
+  final Storage _secureStorage;
```

Comment:
@ducbm051291 @thiennguyen0196 

Should we replace this one with a repository?

Reply:
Yeah, that's correct. @thiennguyen0196, please update this

Reply:
I think we should @ducbm051291 because regarding `Clean Architecture`, use case should have only access to data via repositories, not the service directly 🤔 This is similar to [this](https://github.com/nimblehq/flutter-ic-david-bruce/pull/69#discussion_r1190627533) FYI

Please feel free to correct me if I was wrong 😄

Reply:
Seme legit. Ok, let me update this.